java/elevation-at-point/README.md

```diff
@@ -0,0 +1,37 @@
+# Elevation at point
+
+Get the elevation for a given point on a surface
+
+![Elevation at point app](elevation-at-point.png)
+
+## Use case
+
+Knowing the elevation at a given point in a landscape can aid in navigation, planning and survey in the field.
+
+## How to use the sample
+
+Tap anywhere on the surface to get the elevation at that point.
+
+## How it works
+
+1. Create a `SceneView` and `Scene` with an imagery base map.
+1. Set an `ArcGISTiledElevationService` as the elevation source of the scene's base surface.
```

Comment:
Minor - should this be ArcGISTiledElevationSource?

Reply:
Good spot! Thanks!